src/index.html

```diff
@@ -14,6 +14,7 @@
   <body>
     <!-- W taki sposbu dodajemy fragmenty w główny HTML-plik strony. -->
     <include src="./partials/example.html"></include>
+    <include src="./partials/header.html"></include>
```

Comment:
Marky nie importuj nic do header'a tylko pracuj na swoim branchu i swoim partialu bo później są konflikty

Reply:
ok to usunąć tę 17 linijkę w index.html?